### DISCUSSION TITLE
Changing language

### DISCUSSION THREAD

**Discussion by wojtkens** [bafolts/terraforming-mars#7082] [General]
Hello. There are plenty of completed locales in here https://github.com/terraforming-mars/terraforming-mars/tree/main/src/locales . 
Can I somehow change the language of the game to one of these or I have to wait until it will be integrated into the game? 

Thanks in advance

**Comment by kberg** [comment]
They're already integrated into the game.

**Comment by wojtkens** [comment]
I can only see 6 languages to choose from:
![image](https://github.com/user-attachments/assets/592a048b-43d3-4845-a174-57f3d0dcaf2d)

and I would love to use Polish language which is not there.

**Comment by kberg** [comment]
You are asking about the Steam app. This github us for the open source app. Good luck!

**Comment by wojtkens** [comment]
Ahhh, thanks
